Dockerfile

```diff
@@ -9,15 +8,18 @@ RUN mkdir -p /usr/local/share/jupyter/kernels && chmod -R 777 /usr/local/share/j
 
 ENV JULIA_PATH=/usr/local/julia
 ENV JULIA_DEPOT_PATH=/usr/local/julia
-ENV JULIA_PROJECT=/home/jupyter/.julia/environments/v1.9
+ENV JULIA_PROJECT=/home/jupyter/.julia/environments/askem
 
+RUN mkdir -p /home/jupyter/.julia/environments/askem
```

Comment:
I don't think this needs to exist. I think copy will create the directory structure properly.